code/modules/surgery/anesthesia_machine.dm

```diff
@@ -40,8 +40,29 @@
 	. = ..()
 	if(retract_mask())
 		visible_message("<span class='notice'>[user] retracts the mask back into the [src].</span>")
+	else if(attached_tank)// If attached tank, remove it.
+		attached_tank.forceMove(loc)
+		to_chat(user, "<span class='notice'>You remove the [attached_tank].</span>")
```

Comment:
change all these chat prints to use the span procs rather than how they currently are
like span_notice("example text") that sorta thing

Reply:
knew you would request this